### PR TITLE
fix(account-sidebar): fix account long name are above the other label…

### PR DESCRIPTION
**Pull request by HeyRoach**
…s in account sidebar

### CODE REVIEW

**Comment by wowshakhov** [comment]
Add ellipsis:
selection_022

File: src/style/app.scss [hunk 1]
```diff
@@ -167,6 +167,12 @@ mat-card.custom-card {
   }
 }
 
+.mat-content {
```
[the remark this answers is not in the document]
Comment (HeyRoach) [reply src/style/app.scss]: I use ellipsis, but dots don't appear. I didn't catch why. Is it a problem?

File: src/app/account/account-sidebar/account-users/account-users.component.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.mat-expansion-panel-header {
+  max-width: 100%;
```
Comment (wowshakhov) [inline src/app/account/account-sidebar/account-users/account-users.component.scss]: seems like it doesn't do anything

File: src/app/account/account-sidebar/account-users/account-users.component.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+}
+
+.entity-card-title {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  max-width: 95%;
```
Comment (wowshakhov) [inline src/app/account/account-sidebar/account-users/account-users.component.scss]: is max-width: 95%; necessary?

File: src/app/account/account-sidebar/account-users/account-users.component.scss [hunk 4]
```diff
@@ -0,0 +1,11 @@
+.mat-expansion-panel-header {
+  max-width: 100%;
+}
+
+.entity-card-title {
```
Comment (wowshakhov) [inline src/app/account/account-sidebar/account-users/account-users.component.scss]: you can replace entity-card-title with truncate from app.scss

File: src/app/account/account-sidebar/account-users/account-users.component.html [hunk 5]
```diff
@@ -11,27 +9,20 @@
     (opened)="openItem(user)"
   >
     <mat-expansion-panel-header>
-      <mat-panel-title>
+      <mat-panel-title class="entity-card-title" [matTooltip]="getUserName(user)">
         {{ user.firstname }} {{ user.lastname }}
```
Comment (wowshakhov) [inline src/app/account/account-sidebar/account-users/account-users.component.html]: why do you use a function call in one place and not in the other one? use literal strings in both places or the function call in both places

**Pull request merged**